backend/django_core/static_src/components/page/explore/Index.svelte

```diff
@@ -273,7 +253,10 @@
                                     {@const is_selected = selected_items?.some((item) => item === key)}
 
                                     <button
-                                        on:click|preventDefault={() => update_selected_items(option[0], { key, value })}
+                                        on:click|preventDefault={() => {
+                                            const val = String(value);
+                                            update_selected_items(option[0], { key, val });
+                                        }}
```

Comment:
Does this works?
since we've Array of strings, just pass string and continue filtering?
DO we need this THIS?

Reply:
>Does this works?
since we've Array of strings, just pass string and continue filtering?
DO we need this THIS?

Only you know this